test/auth/AxelarAuthWeighted.js

```diff
@@ -32,13 +32,12 @@ describe('AxelarAuthWeighted', () => {
         wallets = await ethers.getSigners();
 
         owner = wallets[0];
-        operators = sortBy(wallets.slice(3, 9), (wallet) => wallet.address.toLowerCase());
-
+        operators = sortBy(wallets.slice(1, 3), (wallet) => wallet.address.toLowerCase());
         let previousOperatorsLimit = OLD_KEY_RETENTION;
 
-        for (let i = 0; i < wallets.length - 3; i++) {
-            for (let j = i; j < wallets.length - 3; j++) {
-                previousOperators.push(sortBy(wallets.slice(i, j + 3), (wallet) => wallet.address.toLowerCase()));
+        for (let i = 0; i < wallets.length - 2; i++) {
```

Comment:
wallets can have an arbitrary length since it's user dependent. Fix the loop bound explicitly